[DOCS] Update welcome screen to continuous validation (#187)
deepchecks/monitoring: docs/source/getting-started/welcome.rst

.. image:: /_static/images/general/cont_validation_light.png
   :target: https://deepchecks.com/?utm_source=docs.deepchecks.com&utm_medium=referral&utm_campaign=welcome
   :alt: Deepchecks Continuous Validation: Testing, CI & Monitoring
   :align: center
   :width: 80%

========================
Welcome to Deepchecks!
========================

`Deepchecks <https://github.com/deepchecks/deepchecks>`__ is a holistic open-source solution for all of your AI & ML validation needs, 

> enabling to thoroughly test your data and models from research to production.

Reviewer: ```suggestion
enabling you to thoroughly test your data and models from research to production.
```

> See more info in the :ref:`Deepchecks Components for Continuous Validation <welcome__deepchecks_components>`

Reviewer: More info about what? I didn't understand in what cases I should click that

        understanding the results in the system. Follow along the examples or download 
        and run it yourself!

    .. grid-item-card:: 🤖 API Reference 🤖
        :link-type: doc
        :link: /api/index
        
        Reference for all of Deepchecks' SDK's components.

.. _welcome__deepchecks_components:

Deepchecks' Components
=======================

.. grid:: 1
    :gutter: 3

    .. grid-item-card:: Testing Docs
        :link-type: doc
        :link: deepchecks:getting-started/welcome
        :img-top: /_static/images/welcome/research_title.png
        :columns: 4

        Tests during research and model development
    
    .. grid-item-card:: CI Docs
        :link-type: doc
        :link: deepchecks:general/usage/ci_cd
        :img-top: /_static/images/welcome/ci_cd_title.png
        :columns: 4
        
        Tests before deploying the model to production

    .. grid-item-card:: Monitoring Docs (Here)
        :link-type: ref
        :link: welcome__start_with_deepchecks_monitoring
        :img-top: /_static/images/welcome/monitoring_title.png
        :columns: 4

        Tests and continuous monitoring during production
        

> Deepchecks accompanies you through various testing needs such as verifying your data's integrity, 

Reviewer: ```suggestion
Deepchecks is built to help you with your various testing needs such as verifying your data's integrity, 
```
Not sure how to improve this phrasing, but think it should be changed.

inspecting its distributions, validating data splits, evaluating your model and comparing between different models,
throughout the model's lifecycle.

.. image:: /_static/images/general/testing_phases_in_pipeline.png
   :alt: Phases for Continuous Validation of ML Models and Data
   :align: center

|

> Deechecks' continuous validation approach is based on testing the ML models and data throughout the different phases 

Reviewer: ```suggestion
Deechecks' continuous validation approach is based on testing the ML models and data throughout the different development phases 
```
